lib/events/postprocessing/postprocessing.go

```diff
@@ -0,0 +1,80 @@
+/*
+ * Teleport
+ * Copyright (C) 2025  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package postprocessing
+
+import (
+	"context"
+
+	"github.com/gravitational/trace"
+
+	apievents "github.com/gravitational/teleport/api/types/events"
+	"github.com/gravitational/teleport/lib/auth/recordingmetadata"
+	"github.com/gravitational/teleport/lib/auth/summarizer"
+	"github.com/gravitational/teleport/lib/session"
+)
+
+// SessionPostProcessorConfig is the configuration for the session post-processor.
+type SessionPostProcessorConfig struct {
```

Comment:
Thanks for unifying this in a single location! I'm struggling to come up with a good suggestion, naming is _hard_, but `postprocessing.SessionPostProcessorConfig` is slightly redundant and a lot to type.


My first instinct was session.PostProcessingConfig but session is overloaded.